### ISSUE TITLE
Chroma Training produces noise during validation and inference

### ISSUE THREAD

**Issue by mel870** [bghira/SimpleTuner#2211]
Trying to train with Chroma after latest updates. Now the validation images are noise as well as images created during inference per the images attached.  Tried Lokr and Standard Lora with config below. 

Config used:

"--resume_from_checkpoint": "latest",
"--data_backend_config": "/home/Ubuntu/simpletuner-workspace/chroma-test/multidatabackend.json",
"--aspect_bucket_rounding": 2,
"--seed": 42,
"--minimum_image_size": 0,
"--base_model_precision": "no_change"
"--output_dir": "/home/Ubuntu/simpletuner-workspace/output/chroma",
"--pretrained_model_name_or_path": "lodestones/Chroma1-HD",
"--model_family": "chroma",
"--model_flavour": "hd",
"--model_type": "lora",
"--lora_type": "lycoris",
"--lycoris_config": "/home/Ubuntu/simpletuner-workspace/config/lycoris_config.json",
"--max_train_steps": 3000,
"--num_train_epochs": 0,
"--train_batch_size": 2,
"--gradient_checkpointing": "true",
"--caption_dropout_probability": 0.0,
"--mixed_precision": "bf16",
"--optimizer": "adamw_bf16",
"--learning_rate": "1e-3",
"--lr_end": "5e-5",
"--lr_scheduler": "constant_with_warmup",
"--lr_warmup_steps": 100,
"--checkpoint_step_interval": 50,
"--checkpoints_total_limit": 10,
"--validation_seed": 42,
"--validation_step_interval": 50,
"--validation_resolution": "1024x1024",
"--validation_guidance": 4.0,
"--validation_guidance_real": "0.0",
"--validation_num_inference_steps": "28",
"validation_lycoris_strength": 1.0,
"--validation_prompt": "boldanimestyle. A woman with her trusty dog sitting on a fence at dusk, watching over a small herd of cattle. The scene includes a chuck wagon in the background, twinkling stars above, and wildflowers growing along the fence line",
"--flux_guidance_mode": "constant",
"--tokenizer_max_length": 512,
"--evaluation_type": "none",
"--tracker_project_name": "lora-training",
"--tracker_run_name": "simpletuner-lora",
"--report_to": "wandb",
"--validation_torch_compile": "false",
"--disable_benchmark": "false"

Lokr 

"algo": "lokr",
"multiplier": 1.0,
"linear_dim": 10000,
"linear_alpha": 1,
"factor": 12,
"apply_preset": {
"target_module": [
"Attention",
"FeedForward"
],
"module_algo_map": {
"Attention": {
"factor": 12
},
"FeedForward": {
"factor": 6

Dataset:

"cache_dir_text": "cache/text/boldanime",
"cache_dir_vae": "cache/vae/boldanime",
"caption_strategy": "textfile",
"conditioning": [],
"conditioning_data": [],
"crop": false,
"crop_aspect": "square",
"crop_style": "random",
"dataset_type": "image",
"disabled": false,
"hash_filenames": false,
"id": "boldanime",
"instance_data_dir": "/home/Ubuntu/apps/datasets/boldanime/",
"metadata_backend": "discovery",
"preserve_data_backend_cache": false,
"is_regularisation_data": false,
"probability": 1,
"repeats": 1,
"resolution": 1024,
"resolution_type": "pixel_area",
"type": "local",
"video": {}

"cache_dir": "cache/text/boldanime",
"conditioning": [],
"conditioning_data": [],
"crop_aspect": "square",
"crop_style": "random",
"dataset_type": "text_embeds",
"default": true,
"disabled": false,
"hash_filenames": false,
"id": "chroma-test-text-embeds",
"preserve_data_backend_cache": false,
"probability": 1,
"repeats": 0,
"resolution_type": "pixel",
"type": "local",
"video": {},
"write_batch_size": 128

![Image](https://github.com/user-attachments/assets/5c6e1c77-14c0-42b2-bf3e-c6977c8c80c0)
![Image](https://github.com/user-attachments/assets/6229fb4b-e6f5-4a9a-963b-5f6b5eecf20e)
